lib/cron/cache-pools.ts

```diff
@@ -19,6 +19,7 @@ const handler: ScheduledHandler = async (event: EventBridgeEvent<string, void>)
     level: 'info',
     requestId: event.id,
   })
+  setGlobalLogger(log)
```

Comment:
What will this do?

Reply:
it configures the SOR logs to show up in cloudwatch. Before they weren't showing up, pretty sure someone just forgot to add this